ScoutSuite/providers/aws/services/awslambda.py

```diff
@@ -1,34 +1,39 @@
 # -*- coding: utf-8 -*-
-"""
-Lambda-related classes and functions
-"""
-
-from ScoutSuite.providers.aws.configs.regions import RegionalServiceConfig, RegionConfig, api_clients
-
-
-
-########################################
-# LambdaRegionConfig
-########################################
-
-class LambdaRegionConfig(RegionConfig):
-
-    def parse_function(self, global_params, region, function):
+from ScoutSuite.providers.base.configs.resource_config import ResourceConfig
+from ScoutSuite.providers.aws.configs.regions_config import RegionsConfig
+from ScoutSuite.providers.aws.aws_facade import AwsFacade
+from opinel.utils.aws import build_region_list
+
+class LambdaServiceConfig(RegionsConfig):
+    def __init__(self):
+        super(LambdaServiceConfig, self).__init__('lambda')
+
+    async def fetch_all(self, credentials, regions=None, partition_name='aws', targets=None):
```

Comment:
I think you could get rid of `credentials` and `targets`.

Reply:
So true

Reply:
```suggestion
    async def fetch_all(self, regions=None, partition_name='aws'):
```